### DISCUSSION TITLE
文件目录不更新

### DISCUSSION THREAD

**Discussion by Linya-IronMan** [nshen/learn-neovim-lua#41] [Q&A] [answered]
运行 rust 项目，在项目根目录下生成了一个文件，但是在 File Explorer 中找不到。但是通过VSCode可以看到确实是有这个文件的。
应该如何让文件列表实时更新呢？
![image](https://user-images.githubusercontent.com/77786952/174196454-3ac11e27-887d-4810-bd8a-35c72b2865d5.png)

**Comment by Linya-IronMan** [comment, marked as answer]
自问自答，我找到了。  R 就能进行刷新

**Reply by nshen** [reply]
First blood
